src/brackets.js

```diff
@@ -437,7 +438,9 @@ define(function (require, exports, module) {
         }
 
         if (brackets.platform === "win" && !brackets.inBrowser) {
-            PreferencesManager.definePreference("_windowsScrollFix", "boolean", true).on("change", enableOrDisableWinScrollFix);
+            PreferencesManager.definePreference("_windowsScrollFix", "boolean", true, {
+                description: Strings.DESCRIPTION_WINDOW_SCROLL_FIX
```

Comment:
@MarcelGerber Cool. I'll do that. There is another pref that is not used `defaultExtension`, we can exclude that one two.